src/daemons/MetaDaemon.cpp

```diff
@@ -30,9 +30,9 @@ using nebula::Status;
 DEFINE_int32(port, 45500, "Meta daemon listening port");
 DEFINE_bool(reuse_port, true, "Whether to turn on the SO_REUSEPORT option");
 DEFINE_string(data_path, "", "Root data path");
-DEFINE_string(peers, "", "It is a list of IPs split by comma,"
-                         "the ips number equals replica number."
-                         "If empty, it means replica is 1");
+DEFINE_string(meta_server_addrs, "", "It is a list of IPs split by comma,"
+                                     "the ips number equals replica number."
```

Comment:
This line conflicts with the next sentence. If the number of IPs equals to the number of replicas, it means the IP list includes the local IP as well. So when the IP list is empty, the number of replica should be zero

Reply:
Good catch.